src/mcall/mcall.cpp

```diff
@@ -398,6 +405,29 @@ int parse_options(int ac, char * av[]){
 			configFile 				<<	options[k].as<int>();
 			cout 					<<	options[k].as<int>();
 		}
+        else if(k == "statSplit"){
+            opts.statSplit          =   1; // Split stats marker
+            string inputChromSplit = options[k].as<string>(); // Get the input string
+            char *iCSc = new char[inputChromSplit.length()+1];
+            strcpy(iCSc, inputChromSplit.c_str());
+            //const char *groupspliter = ";";
+            //const char *chromspliter = ",";
+            // Split the string and store all input chromsomes from chromArr1, chromArr2 into opts.splitChrom1 and opts.splitChrom2    
+	    char *chromArr1 = strtok(iCSc, "/");
```

Comment:
Consider to use boost::split() in a C++ style, and we have the whole string stored anyway. Such as,

```
vector <string> toMergeFiles;
boost::split(toMergeFiles, opts.ratiosFiles[i], boost::is_any_of(","));
```